src/Angor/Client/Pages/Browse.razor

```diff
@@ -215,11 +216,93 @@ else
     protected override async Task OnInitializedAsync()
     {
         projects = SessionStorage.GetProjectIndexerData() ?? new List<ProjectIndexerData>();
-        if (!projects.Any())
+
+        if (projects.Any())
+        {
+            // Projects exist, initialize the search focus
+            await JS.InvokeVoidAsync("angor.initSearchFocus", "searchQuery");
```

Comment:
is there no way to focus with blazor code?
This will just put the focus on the search box right?

Reply:
It is possible, but using js has better performance